### ISSUE TITLE
[Request] Add option to detect last chapter for Upnext popup

### ISSUE THREAD

**Issue by psp2111-ADSLGATE** [MoojMidge/service.upnext#21]
if it possible will you add an option to detect chapters/bookmarks in videos files, and use it as indicator for Next episode popup.

**Comment by MoojMidge** [comment]
I don't believe Kodi offers any method for a plugin to directly access this information.

Some information is partially exposed via infobools that are typically used by a skin, and this could theoretically also be used by a plugin, but I have never looked into it.

Are you encountering many scenarios where the detector does not work reliably?

**Comment by psp2111-ADSLGATE** [comment]
weird I have 3rd party addon installed with this feature and can be enabled with its built-in next episode popup. 
if you want to look into it [here](https://github.com/Tikipeter/repository.tikipeter/tree/main/zips/plugin.video.fen)

Regular timed detector works perfectly fine. but it really hard to tune it with every show.
for "Automatic end credits detection" I really dont know how it work exactly and enable it dose not seems to change anything?!

using 3rd party addon players with TMDbH.

**Comment by psp2111-ADSLGATE** [comment]
Mainly I use my PC Windows 11 with Nvidia GPU.
So to use Automatic end credits detection I need to disable hardware acceleration?

>Yeah that uses the infobools I was talking about
is it easy to add it or too much work and headache?

**Comment by MoojMidge** [comment]
> Mainly I use my PC Windows 11 with Nvidia GPU.
> So to use Automatic end credits detection I need to disable hardware acceleration?

I don't have the same/similar hardware but it should work with HW acceleration enabled. If it doesn't work enable debug and detector debug logging in UpNext, send it through and I can have a look.


> is it easy to add it or too much work and headache?

Shouldn't be too difficult, but no time at the moment.
